src/routes/profile.router.js

```diff
@@ -0,0 +1,172 @@
+const { User } = require('../models/User.model');
+const { ApiError } = require('../exceptions/api.error');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
The import statement for `User` should reference the correct path. Ensure that the path './models/User.model' is correct and matches the actual file location.